### DISCUSSION TITLE
Name part cardinality change

### DISCUSSION THREAD

**Discussion by tychonievich** [FamilySearch/GEDCOM#146] [Announcements]
The steering committee has made a change to correct an ambiguity in 7.0.0 through 7.0.7: we've changed the cardinality of `GIVN` and other name parts from `{0:M}` to `{0:1}`, as it was under 5.5.1 and earlier (#143). Previously the multiple cardinalities were not in the changelog for 7.0.0 and no semantics for them was defined.

We believe that this is permitted as a patch under the clause

> If there is an ambiguity or contradiction in the specification, it will be resolved in a patch version unless it is known that implementations interpreted the spec differently and that clarifying the intended meaning would cause incompatibilities between those implementations.

If you (know of) an application that implemented multiple copies of the same name part substructure in their 7.0 tool, please let us know.

**Comment by dthaler** [comment]
issue #152 covered this and it was resolved in PR #156.  So we believe this is done now. If there is follow-up discussion, can do so in issue 152
